### PR TITLE
Configuring the python model tests to execute on Dataproc cluster instead of serverless

### PR DESCRIPTION
**Pull request by nssalian**
resolves #

### Description

Configuring the python model tests to execute on Dataproc cluster instead of serverless

### Checklist

- [x] I have read [the contributing guide](https://github.com/dbt-labs/dbt-bigquery/blob/main/CONTRIBUTING.md) and understand what's expected of me
- [x] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [x] I have run this code in development and it appears to resolve the stated issue
- [x] This PR includes tests, or tests are not required/relevant for this PR
- [ ] I have [opened an issue to add/update docs](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose), or docs changes are not required/relevant for this PR
- [ ] I have run `changie new` to [create a changelog entry](https://github.com/dbt-labs/dbt-bigquery/blob/main/CONTRIBUTING.md#Adding-CHANGELOG-Entry)

### CODE REVIEW

File: tests/functional/adapter/test_python_model.py [hunk 1]
```diff
@@ -3,16 +3,20 @@
 from dbt.tests.util import run_dbt, write_file
 import dbt.tests.adapter.python_model.test_python_model as dbt_tests
 
-TEST_SKIP_MESSAGE = "Skipping the Tests since Dataproc serverless is not stable. " \
-                    "TODO: Fix later"
+
+@pytest.fixture(scope="module")
```
Comment (colin-rogers-dbt) [inline tests/functional/adapter/test_python_model.py:7]: Super bizarre that this also fails on serverless, have we tried the others on cluster mode?

Comment (nssalian) [reply tests/functional/adapter/test_python_model.py:7]: They all work on cluster mode. @ChenyuLInx tested this as well. 2 out of 3 working isn't giving confidence here.

Comment (ChenyuLInx) [reply tests/functional/adapter/test_python_model.py:7]: Yes locally on cluster all tests passed. The failure is likely due to Dataproc not dbt.

Comment (nssalian) [reply tests/functional/adapter/test_python_model.py:7]: Re-running to make sure this isn't a transient error.

**Review by ChenyuLInx — COMMENTED** [review]
Looks good to me! Tested this change locally and it works. Dataproc is not very stable

**Comment by nssalian** [comment]
@ChenyuLInx , @colin-rogers-dbt , I don't think it is wise to push this up unless the Dataproc instability is resolved. I'm open to keeping this as pending work for later.

**Pull request closed without merging**